### PR TITLE
Add PiP support on Chrome for Android

### PR DESCRIPTION
**Pull request by beaufortfrancois**
With upcoming Picture-in-Picture support coming on Chrome for Android, this PR updates the article content and its title which was incorrect due to https://github.com/GoogleChrome/developer.chrome.com/pull/2321.

Live preview: https://deploy-preview-3273--developer-chrome-com.netlify.app/blog/watch-video-using-picture-in-picture/

FYI @liberato-at-chromium

I'm updating MDN compat data at https://github.com/mdn/browser-compat-data/pull/17047 as well.

### CODE REVIEW

File: site/en/blog/watch-video-using-picture-in-picture/index.md [hunk 1]
```diff
@@ -349,8 +349,13 @@ API are currently implemented in Chrome and other browsers.
 
 Here's what you can expect to see in the near future:
 
-- Picture-in-Picture will be supported in Android O.
 - Web developers will be able to [add custom Picture-in-Picture controls].
+- A [new Web API] will be provided to display arbitrary `HTMLElement`s in a floating window.
```
Comment (jpmedley) [inline site/en/blog/watch-video-using-picture-in-picture/index.md:353]:
```suggestion
- A [new Web API] will be provided to display arbitrary `HTMLElement` objects in a floating window.
```

File: site/en/blog/watch-video-using-picture-in-picture/index.md [hunk 2]
```diff
@@ -395,3 +400,5 @@ involved in the [standardization effort].
 [jennifer apacible]: https://twitter.com/japacible
 [media session api]: /media-session
 [audio playlist sample]: https://googlechrome.github.io/samples/picture-in-picture/audio-playlist
+[new web api]: https://github.com/steimelchrome/document-pip-explainer/blob/main/explainer.md
+[mdn]: https://developer.mozilla.org/docs/Web/API/Picture-in-Picture_API#browser_compatibility
```
Comment (jpmedley) [inline site/en/blog/watch-video-using-picture-in-picture/index.md:404]:
```suggestion
```

File: site/en/blog/watch-video-using-picture-in-picture/index.md [hunk 3]
```diff
@@ -18,7 +18,7 @@ Picture-in-Picture (PiP) allows users to watch videos in a floating window
 (always on top of other windows) so they can keep an eye on what they’re
 watching while interacting with other sites, or applications.
 
-With the new [Picture-in-Picture Web API], you can initiate and control
+With the [Picture-in-Picture Web API], you can initiate and control
```
Comment (jpmedley) [inline site/en/blog/watch-video-using-picture-in-picture/index.md:21]:
```suggestion
With the [Picture-in-Picture Web API], you can start and control
```

Comment (jpmedley) [reply site/en/blog/watch-video-using-picture-in-picture/index.md:21]: A best practice in English writing is to use simpler words when they're available.

File: site/en/blog/watch-video-using-picture-in-picture/index.md [hunk 4]
```diff
@@ -349,8 +349,13 @@ API are currently implemented in Chrome and other browsers.
 
 Here's what you can expect to see in the near future:
 
-- Picture-in-Picture will be supported in Android O.
 - Web developers will be able to [add custom Picture-in-Picture controls].
+- A [new Web API] will be provided to display arbitrary `HTMLElement`s in a floating window.
+
+## Browser support
+
+The Picture-in-Picture Web API is supported in Chrome, Edge, Opera, and Safari.
```
[the remark this answers is not in the document]
Comment (beaufortfrancois) [reply site/en/blog/watch-video-using-picture-in-picture/index.md:357]: `BrowserCompat` is not implemented in developer.chrome.com. That's why I linked to mdn instead and specified explicit support. This has 2 advantages:
1. people who don't click on mdn can see browser support
2. search engines can give short answers (seo)

File: site/en/blog/watch-video-using-picture-in-picture/index.md (hunk 3 above)
Comment (beaufortfrancois) [reply site/en/blog/watch-video-using-picture-in-picture/index.md:21]: Thanks!

File: site/en/blog/watch-video-using-picture-in-picture/index.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (beaufortfrancois) [reply site/en/blog/watch-video-using-picture-in-picture/index.md:357]: See https://chat.google.com/room/AAAAPcdcIP0/hDGJR_APjQQ

**Comment by beaufortfrancois** [comment]
@jpmedley Do my comments look good to you?

**Review by jpmedley — APPROVED** [review]

**Pull request merged**